src/Symfony/Component/Workflow/EventListener/GuardExpression.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Symfony\Component\Workflow\EventListener;
+
+use Symfony\Component\Workflow\Transition;
+
+class GuardExpression
+{
+
+    /**
+     * @var Transition
+     */
+    private $transition;
+
+    /**
+     * @var string
+     */
+    private $expression;
+
+    /**
+     * @return Transition
+     */
+    public function getTransition(): Transition
+    {
+        return $this->transition;
+    }
+
+    /**
+     * @return string
+     */
+    public function getExpression(): string
+    {
+        return $this->expression;
+    }
+
+    /**
+     * GuardConfiguration constructor.
+     * @param Transition $transition
+     * @param string $expression
+     */
+    public function __construct(Transition $transition, string $expression)
+    {
+        $this->transition = $transition;
+        $this->expression = $expression;
+    }
+}
```

Comment:
missing EOL. Also please remove all docblocks in this class: they don't provide anything on top of the existing type declaration, and our policy is to not add them in this situation.